mailbox/api/src/main/java/org/apache/james/mailbox/events/EventDeadLettersHealthCheck.java

```diff
@@ -54,8 +51,7 @@ public Result check() {
 
             return Result.healthy(COMPONENT_NAME);
         } catch (Exception e) {
-            LOGGER.error("Error checking EventDeadLettersHealthCheck", e);
-            return Result.unhealthy(COMPONENT_NAME, e.getMessage());
+            return Result.unhealthy(COMPONENT_NAME, e.getMessage(), e);
```

Comment:
return Result.unhealthy(COMPONENT_NAME, "Error checking EventDeadLettersHealthCheck", e);

Having 2 time the exception message have no purposes